docs/workforce-management-framework/upstream-1/running-the-published-repositories.adoc

```diff
@@ -1,22 +1,24 @@
 include::topics/shared/attributes.adoc[]
 
-:chapter: running-the-published-repositories
+//':context:' is a vital parameter. See: http://asciidoctor.org/docs/user-manual/#include-multiple
+:context: running-the-published-repositories
+
+= Running the Demo App from the Published Repositories
 
-ifdef::upstream[]
-[discrete]
-endif::upstream[]
-= Running the Published Repositories for the Demo App
```

Comment:
I'm going to add a variable here so you can call them what you want when you/Eng/Product have made a decision on what to call them.